doc/cug.tex

```diff
@@ -2822,9 +2822,13 @@ \subsubsection{Graph Types}
             graph = "delayed_cold_corge => corge"
 
         # Repeat once at the final cycle point
-        [[[ R1/P0Y ]]]
+        [[[ R1/$ ]]]
```

Comment:
We should retain documentation for the implicit form as well (as it is still supported).

In fact, for completeness we should probably document the various ways of getting the same result, and maybe point out which recurrence form each conforms to (Rn/INTVL/STOP, Rn/START/INTVL), for the new explicit notation and the existing implicit forms.  In this case:
 * `R1/P0Y`  
 * `R1/P0Y/$`
 * `R1/$`
 
It can get confusing, especially when you consider cases where dropping an explicit `$` gives a different result by turning a point into an interval - `R2/$-P1Y/P1Y` to `R2/-P1Y/P1Y` ...